cppcache/CMakeLists.txt

```diff
@@ -70,11 +70,14 @@ if(CMAKE_SYSTEM_NAME STREQUAL "SunOS")
     demangle
     kstat
   )
-endif()
-if (${WIN32})
+elseif (WIN32)
   target_link_libraries(_apache-geode INTERFACE
     Dbghelp
   )
+  target_compile_definitions(_apache-geode INTERFACE
+    # Required for Boost.WinAPI
+    _WIN32_WINNT=0x06020000
```

Comment:
Whoa, what's going on here?!

Reply:
Some Boost dark magic! I couldn’t find any other good place to add it. It’s required to set the minimum NT version Boost.WinAPI will compile for. In this case NT 6.2.

Reply:
It popped up with the Boost 1.66